src/commands/paint.ts

```diff
@@ -97,15 +97,19 @@ export function paint(
     process.stdout.write(`${chalk.bold('Snowpack')}\n\n`);
     // Dashboard
     if (devMode) {
-      process.stdout.write(
-        `  ${chalk.bold.cyan(`${devMode.protocol}//localhost:${devMode.port}`)}`,
-      );
-      for (const ip of devMode.ips) {
-        process.stdout.write(
-          `${chalk.cyan(` > `)}${chalk.bold.cyan(`${devMode.protocol}//${ip}:${devMode.port}`)}`,
-        );
+      // SERVER_START event happened
+      if (startTimeMs > 0 && port > 0 && protocol) {
+        process.stdout.write(`  ${chalk.bold.cyan(`${protocol}//localhost:${port}`)}`);
+        for (const ip of ips) {
+          process.stdout.write(
+            `${chalk.cyan(` > `)}${chalk.bold.cyan(`${protocol}//${ip}:${port}`)}`,
+          );
+        }
+        process.stdout.write('\n' + chalk.dim(`  Server started in ${startTimeMs}ms.\n\n`));
```

Comment:
In a couple of places, we do handling logic to reset the server start time if the user was prompted, a re-install ran, etc.

```
// We do this in a few places to reset the timer after a user-prompt
serverStart = Date.now();
```

What do you think about instead removing these special resets and putting some better handling in the paint function, something like:

```
if (startTimeMs > 1000) {
  process.stdout.write('\n' + chalk.dim(`  Server started in ${startTimeMs}ms.\n\n`));
} else {
  process.stdout.write('\n' + chalk.dim(`  Server started.\n\n`));
}
```

Not in an attempt to hide slow startups, but in an attempt to remove the need to custom handle every user prompt, re-install, etc.

Reply:
Not pressing, feel free to resolve in a follow-up PR or ignore

Reply:
That makes sense to me. For navigations and such. I’m assuming that should be a `< 1000` above right? 👆

Reply:
lol yes! it could even be lower, like 500ms. Up to you!

Reply:
`1000` works for now